roles/deploy/tasks/prepare_database.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Ensure the database is created, migrated and seeded (first deployment only)
+
+- name: load database schema
+  command: bash -lc "bundle exec rake db:schema:load RAILS_ENV={{ rails_env }} I_AM_SURE=1"
+  args:
+    chdir: "{{ build_path }}"
+  tags:
+    - rake
+    - skip_ansible_lint
+
+- name: migrate database
+  command: bash -lc "bundle exec rake db:migrate RAILS_ENV={{ rails_env }}"
+  args:
+    chdir: "{{ build_path }}"
+  tags:
+    - rake
+    - skip_ansible_lint
+  notify: restart puma
+
+- name: seed database
+  command: bash -lc "bundle exec rake db:seed RAILS_ENV={{ rails_env }}"
+  args:
+    chdir: "{{ build_path }}"
+  tags:
+    - seed
+    - skip_ansible_lint
+  notify: restart puma
```

Comment:
I think that we can replace these three commands with one:
```
rails db:setup # Creates all databases, loads all schemas, and initializes with the seed
```

There shouldn't be any pending migrations but if there were then the migration step is run after the asset compilation anyway.

Reply:
This sounds like a good idea, and will be faster with less ansible commands and one less rake task.

But more importantly, it means less code, which will remove the need for a new file :D

Reply:
Hmmm that sounds nice and simple, but I'm trying to figure out if we can remove that `I_AM_SURE=1` on line 5, or why it's there in the first place. Ironically I am not sure what it does...

Reply:
Ah I see it's related to the `rails_safe_tasks` gem :+1:

Reply:
It protects from overriding a production db, right? So, we don't actually want that in our deployment. We only want to do safe things during a normal deploy.

Reply:
> We only want to do safe things during a normal deploy.

I think that's a very good point! We do have a safety check beforehand, but it's checking for a specific table, so not foolproof (eg what if we removed the table one day?).

I agree and think that we should remove ` I_AM_SURE=1` from `deploy.yml`, so I think will open a new PR.

Reply:
Indeed, I think the `db:setup` is non-destructive (it only creates if missing). In that case, the `I_AM_SURE=1` confirmation should have no effect. I don't know how the old `rails_safe_tasks` gem handles this, but in any case this isn't dangerous so I won't bother updating it for now.